tests/models.py

```diff
@@ -18,7 +18,7 @@ class AllFields(models.Model):
     datetime = models.DateTimeField()
     decimal = models.DecimalField(decimal_places=2, max_digits=4)
     email = models.EmailField()
-    file_path = models.FilePathField()
+    file_path = models.FilePathField(path="tests")
```

Comment:
I _believe_ this is a `path=None` => `path=''` defaults issue in the django form cleanup stuff. But I'm also OK with putting in this change without 100% confidence just because contextually it doesn't matter much for this test